.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,7 @@
 contact_links:
+  - name: "\U0001F525 Suggest a Fireside Chat"
+    url: https://github.com/the-turing-way/fireside-chats/issues
```

Comment:
This just goes to the main issues page on the fireside chat repo - is there an issue template on that repo that we should be linking to (or should we make one?)